resources/extruders/vertex_nano_k8600_extruder_0.def.json

```diff
@@ -0,0 +1,16 @@
+{
+    "id": "vertex_nano_k8600_extruder_0",
+    "version": 1,
+    "name": "Extruder 1",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "vertex_nano_k8600",
+        "position": "0"
+    },
+
+    "overrides": {
+        "extruder_nr": { "default_value": 0 },
+        "machine_nozzle_size": { "default_value": 0.35 },
```

Comment:
Is the nozzle size actually 0.35mm or did you intend to have a line width of 0.35mm? Not a problem here really but I'm asking in case you misunderstood this.

Reply:
Nozzle is actually 0.35